### PR TITLE
{Compute} Update history notes

### PR DESCRIPTION
**Pull request by qwordy**
**Description**

An user reported "virtual_machine_extension_type" becomes "type_properties_type" in VM resource

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

File: src/azure-cli/HISTORY.rst [hunk 1]
```diff
@@ -364,7 +364,7 @@ Release History
 **Compute**
 
 * New license type RHEL_ELS_6 (#16012)
-* Adopt track2 SDK, azure-mgmt-compute==18.0.0 (#15750)
+* Adopt track2 SDK, azure-mgmt-compute==18.0.0, including breaking changes such as property name change (#15750)
```
Comment (jiasli) [inline src/azure-cli/HISTORY.rst:367]: Shouldn't we explicitly call out what BREAKING CHANGES are introduced?

Comment (qwordy) [inline src/azure-cli/HISTORY.rst:367]:
```suggestion
* Adopt track2 SDK, azure-mgmt-compute==18.0.0 (#15750)
* [BREAKING CHANGE] Property names change due to track2 SDK. For example, "virtual_machine_extension_type" becomes "type_properties_type" in VM resource.
```

**Review by fengzhou-msft — APPROVED** [review]

File: src/azure-cli/HISTORY.rst (hunk 1 above)
Comment (qwordy) [reply src/azure-cli/HISTORY.rst]: Done

**Pull request merged**